libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx

```diff
@@ -13,7 +13,12 @@ export type HomeTabFeaturedProps = {
 
 function HomeTabFeatured(props:HomeTabFeaturedProps) {
   const themeFilter = useContext(ThemeContext)
-
+  const handleStartLearneth = async () => {
+    await props.plugin.appManager.activatePlugin(['LearnEth', 'solidityUnitTesting'])
+    props.plugin.verticalIcons.select('LearnEth')
+    _paq.push(['trackEvent', 'hometab', 'featuredSection', 'LearnEth'])
+    await props.plugin.call('LearnEth', 'home')
```

Comment:
I don't think `home` is a method for learneth. Clicking on button shows this error in browser console.
```
engine.ts:132 Uncaught (in promise) Error: Cannot call method "home" of "LearnEth" from "home", because "home" is not exposed. Here is the list of exposed methods: "startTutorial","addRepository","canDeactivate"
    at RemixEngine.<anonymous> (engine.ts:132:13)
    at Generator.next (<anonymous>)
    at fulfilled (tslib.es6.js:73:43)
```